config-linux.md

```diff
@@ -637,6 +637,30 @@ The following parameters can be specified to set up seccomp:
     "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
 ```
 
+### <a name="configLinuxRdmaLimits" />RDMA resource limits
+
+**`rdma_limits`** (array of objects, OPTIONAL) represents the `rdma` controller which allows the limiting of the
+resources related to network interfaces which support RDMA (e.g. HCA handles).
+For more information, see the RDMA section of [the kernel cgroups documentation][cgroup-v1-rdma].
+
+Each entry has the following structure:
+
+* **`interface_name`** *(string, REQUIRED)* - a name of an interface
```

Comment:
`config_linux.md` isn't consistent on this, but `config.md` is pretty consistent in not wrapping the type/requirement parenthetical in `*`.  That would make this line:

    * **`interface_name`** (string, REQUIRED) - a name of an interface

Because of the current `config-linux.md` inconsistency, I'd be fine with you using wrapping `*` or not in this commit, although you should probably be internally consistent.  Can you drop them here and below, or keep them here and in the `rdma_limits` entry above?